[{
	"raw": {},
	"required": [
		"Filename",
		"Size"
	],
	"package": "python-faulthandler",
	"source": "faulthandler",
	"version": "2.4-1",
	"section": "universe/python",
	"priority": "optional",
	"architecture": "amd64",
	"depends": [
		"python (<< 2.8)",
		"python (>= 2.7~)",
		"python:any (<< 2.8)",
		"python:any (>= 2.7~)",
		"libc6 (>= 2.4)"
	],
	"installedSize": 44,
	"maintainer": "Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>",
	"description": "Display the Python backtrace on a crash",
	"homepage": "https://faulthandler.readthedocs.io/",
	"filename": "pool/universe/f/faulthandler/python-faulthandler_2.4-1_amd64.deb",
	"size": 12450,
	"md5": "cb4a29cf40f0ab5419cba450c24fbb7d",
	"sha1": "e8313c953ba5bdcf13ae2b34c1442d6654882d18",
	"sha256": "7e7ec78d44dfd56cb356a3310fc7b3f4dbd3e84b19bbb684046574296564991c",
	"descriptionMd5": "bd1a3d7fdc686b27cdcbd4b2a7946734"
}]